### DISCUSSION TITLE
I cannot get accessibility to my account.

### DISCUSSION THREAD

**Discussion by akerfelter** [community/community#28482] [Other Feature Feedback, Questions, & Ideas]
Hello everyone I hope you all  to have very good time in your lives.
I see that now my account has been flagged, so I cannot have access to it anymore, and have been hidden from the public. The reason for been flagged is spam. However; I did not have any suspicious activity so far, but make sure that I have never had a suspicious activity, and there was a mistake.

**Comment by cehfisher** [comment]
Hi @akerfelter - I hope you were able to resolve your issue. It looks like it was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!

**Reply by queenofcorgis** [reply]
Done! As a reminder, the Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
